### DISCUSSION TITLE
openstreetmap does not work

### DISCUSSION THREAD

**Discussion by Benleie** [tomickigrzegorz/leaflet-examples#69] [Announcements]
Failed to load resource from openstreetmap
![image](https://github.com/tomickigrzegorz/leaflet-examples/assets/24362871/5eb35ec3-2799-4de7-b4f7-f2fa972abd70)

**Comment by tomickigrzegorz** [comment]
I checked myself and everything works fine
![image](https://github.com/tomickigrzegorz/leaflet-examples/assets/25438490/b890767d-b5fa-41d0-a91d-faabdc498b60)

**Comment by tomickigrzegorz** [comment]
@Benleie The question is can you see this tile https://tile.openstreetmap.org/18/146371/86316.png
If not, the second question is which country are you working from?

**Reply by Benleie** [reply]
Thank you for your quick and detailed reply. Now I can see this tile image and run the example normally, after i adjusted my network and VPN config.  I am from China. Thanks!

**Reply by tomickigrzegorz** [reply]
Well, you have the answer 😀
